package.json

```diff
@@ -10,8 +10,8 @@
     "lint": "ionic-app-scripts lint",
     "ionic:build": "ionic-app-scripts build",
     "ionic:serve": "ionic-app-scripts serve",
-    "ionic:android": "ionic cordova run android --emulator --no-fetch",
-    "ionic:ios": "ionic cordova run ios --emulator --no-fetch",
```

Comment:
is there a reason we want to remove --emulator by default? seems pointless as most people will run it using an emulator for quick dev

Reply:
nevermind, @ciaranRoche has informed me it looks for an emulator if there is no device present. kudos!